mods/default/trees.lua

```diff
@@ -604,3 +529,145 @@ function default.sapling_on_place(itemstack, placer, pointed_thing,
 
 	return itemstack
 end
+
+-- Grow sapling
+
+default.sapling_growth_defs = {}
```

Comment:
If you need this public, please document it in `game_api.txt`. Otherwise it should be made private to avoid mods relying on undocumented behaviour.